src/biokbase/narrative/tests/test_widgetmanager.py

```diff
@@ -190,6 +190,36 @@ def test_infer_upas_simple_widget(self):
         self.assertIsInstance(upas, dict)
         self.assertFalse(upas)
 
+    @mock.patch('biokbase.narrative.widgetmanager.clients.get', get_mock_client)
```

Comment:
I don't really understand the code here that well, so my review might not be worth much.